<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-24T15:07:18 EET" name="Default suite" finished-at="2022-01-24T15:07:25 EET" duration-ms="6916">
    <groups>
    </groups>
    <test started-at="2022-01-24T15:07:18 EET" name="Default test" finished-at="2022-01-24T15:07:25 EET" duration-ms="6916">
      <class name="testScripts.TemperatureTest">
        <test-method is-config="true" signature="beforeClassSetUp(java.lang.String)[pri:0, instance:testScripts.TemperatureTest@6e0dec4a]" started-at="2022-01-24T15:07:18 EET" name="beforeClassSetUp" finished-at="2022-01-24T15:07:24 EET" duration-ms="5245" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClassSetUp -->
        <test-method is-config="true" signature="calling()[pri:0, instance:testScripts.TemperatureTest@6e0dec4a]" started-at="2022-01-24T15:07:24 EET" name="calling" finished-at="2022-01-24T15:07:24 EET" duration-ms="29" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- calling -->
        <test-method signature="temperature()[pri:0, instance:testScripts.TemperatureTest@6e0dec4a]" started-at="2022-01-24T15:07:24 EET" name="temperature" data-provider="Data.xlsx" finished-at="2022-01-24T15:07:24 EET" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void testScripts.TemperatureTest.temperature() throws java.lang.InterruptedException requires a @DataProvider named : Data.xlsx]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void testScripts.TemperatureTest.temperature() throws java.lang.InterruptedException requires a @DataProvider named : Data.xlsx
at org.testng.internal.Parameters.findDataProvider(Parameters.java:563)
at org.testng.internal.Parameters.handleParameters(Parameters.java:776)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- temperature -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:testScripts.TemperatureTest@6e0dec4a]" started-at="2022-01-24T15:07:24 EET" name="afterClass" finished-at="2022-01-24T15:07:25 EET" duration-ms="1581" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- testScripts.TemperatureTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
